mavros/src/plugins/local_position.cpp

```diff
@@ -46,7 +47,8 @@ class LocalPositionPlugin : public MavRosPlugin {
 		lp_nh.param<std::string>("tf/frame_id", tf_frame_id, "local_origin");
 		lp_nh.param<std::string>("tf/child_frame_id", tf_child_frame_id, "fcu");
 
-		local_position = lp_nh.advertise<geometry_msgs::PoseStamped>("local", 10);
+		local_position = lp_nh.advertise<geometry_msgs::PoseStamped>("local/position", 10);
+		local_velocity = lp_nh.advertise<geometry_msgs::TwistStamped>("local/velocity", 10);
```

Comment:
Nope, changing topic names usually return with lots questions. While sometime that is absolutely required, but now i'd prefer only addition.

I think "velocity" is enough (`~local_position/velocity`).